code/mob.dm

```diff
@@ -3026,3 +3026,7 @@
 		..(0)
 	else
 		..(newdensity)
+
+// to check if someone is abusing cameras with stuff like artifacts, power gloves, etc
+/mob/proc/in_real_view_range(var/turf/T)
+	return src.client && IN_RANGE(T, src, (istext(src.client.view) ? 10 : src.client.view))
```

Comment:
What is 10

Reply:
The default view range that is used for people that somehow don't have clients, I guess.
I could put it into a define, I guess, it's not really used anywhere else though, so I guess I'd put it here?

Reply:
Why isn't this the horizontal screen size?

Reply:
I suppose whoever wrote the original artifact code didn't feel like extracting it from the string or something, I didn't give it that much thought, I guess I could make it less dumb though.

Reply:
I made it get the actual larger measurement of length and width now, it feels a little overengineered, but idk.

Reply:
> I made it get the actual larger measurement of length and width now, it feels a little overengineered, but idk.

It's already a define

Reply:
Ah, I see,, thanks, I will change it to that as soon as I get to it!

Reply:
ok, I just use the widescreen width now, I guess that's fair for everyone